src/commands/aksKaito/aksKaitoManage.ts

```diff
@@ -0,0 +1,109 @@
+        return;
+    }
+
+    const clusterNode = getAksClusterTreeNode(target, cloudExplorer);
+    if (failed(clusterNode)) {
+        vscode.window.showErrorMessage(clusterNode.error);
+        return;
+    }
+
+    const extension = getExtension();
+    if (failed(extension)) {
+        vscode.window.showErrorMessage(extension.error);
+        return;
+    }
+
+    if (!kubectl.available) {
+        vscode.window.showWarningMessage(`Kubectl is unavailable.`);
+        return;
+    }
+
+    if (!cloudExplorer.available) {
+        vscode.window.showWarningMessage(`Cloud explorer is unavailable.`);
+        return;
+    }
+
+    if (!clusterExplorer.available) {
+        vscode.window.showWarningMessage(`Cluster explorer is unavailable.`);
+        return;
+    }
+    const { name: clusterName, armId, subscriptionId, resourceGroupName } = clusterNode.result;
+    // "kaito-" is assuming kaito pods are still named kaito-workspace & kaito-gpu-provisioner
+    const filterKaitoPodNames = await longRunning(`Checking if KAITO is installed.`, () => {
+        return filterPodName(sessionProvider.result, kubectl, subscriptionId, resourceGroupName, clusterName, "kaito-");
+    });
+
+    if (failed(filterKaitoPodNames)) {
+        vscode.window.showErrorMessage(filterKaitoPodNames.error);
+        return;
+    }
+
+    if (filterKaitoPodNames.result.length === 0) {
+        vscode.window.showWarningMessage(
+            `Please install Kaito for cluster ${clusterName}. \n \n Kaito Workspace generation is only enabled when kaito is installed. Skipping generation.`,
+        );
+        return;
+    }
+    const clusterInfo = await getKubernetesClusterInfo(sessionProvider.result, target, cloudExplorer, clusterExplorer);
+    if (failed(clusterInfo)) {
+        vscode.window.showErrorMessage(clusterInfo.error);
+        return;
+    }
+
+    // The logic below is to acquire the initial deployment data.
+    const kubeConfigFile = await tmpfile.createTempFile(clusterInfo.result.kubeconfigYaml, "yaml");
+    const command = `get workspace -o json`;
+    const kubectlresult = await invokeKubectlCommand(kubectl, kubeConfigFile.filePath, command);
+    if (failed(kubectlresult)) {
+        vscode.window.showErrorMessage(`Error retrieving workspaces: ${kubectlresult.error}`);
+        return;
+    }
+    const models = [];
+    const data = JSON.parse(kubectlresult.result.stdout);
+    for (const item of data.items) {
+        const conditions: Array<{ type: string; status: string }> = item.status?.conditions || [];
+        const { resourceReady, inferenceReady, workspaceReady } = getConditions(conditions);
+        models.push({
+            name: item.inference?.preset?.name,
+            instance: item.resource?.instanceType,
+            resourceReady: resourceReady,
+            inferenceReady: inferenceReady,
+            workspaceReady: workspaceReady,
+            age: convertAgeToMinutes(item.metadata?.creationTimestamp),
```

Comment:
💡 Suggestion if there is a way might be worth to comment on this model, and where the age will get used.